jdaviz/configs/default/plugins/subset_tools/subset_tools.py

```diff
@@ -211,42 +212,91 @@ def get_regions(self, region_type=None, list_of_subset_labels=None,
+            The default behavior (None) will return Pixel/Sky region based on app link type
+            (Sky for Cubeviz), using the WCS of the subset's parent dataset (i.e the data
+            layer the subset was created on).
 
         Returns
         -------
         regions : dict
             A dictionary mapping subset labels to their respective ``regions``
             objects (for spatial regions) or ``SpectralRegions`` objects
             (for spectral regions).
+
+        Examples
+        --------
+        >>> from jdaviz import Imviz, Cubeviz
+        >>> from regions import PixCoord, CirclePixelRegion, CircleSkyRegion
+        >>> from astropy.nddata import NDData
+        >>> import numpy as np
+        >>> import astropy.units as u
+        >>> imviz = Imviz()
+        >>> imviz.link_data(align_by='pixels')
+        >>> data = NDData(np.ones((128, 128)) * u.nJy, wcs=getfixture('image_2d_wcs'))
+        >>> imviz.load_data(data)
+        >>> plg = imviz.plugins['Subset Tools']
+        >>> plg.import_region(CirclePixelRegion(center=PixCoord(x=1163.618408203125, y=1433.47998046875), radius=141.28575134277344))  # noqa E501
+        >>> type(plg.get_regions()['Subset 1'])
+        <class 'regions.shapes.circle.CirclePixelRegion'>
+        >>> type(plg.get_regions(wrt_data='NDData[DATA]')['Subset 1'])
+        <class 'regions.shapes.circle.CircleSkyRegion'>
+        >>> imviz.app.delete_subsets()
+        >>> imviz.link_data(align_by='wcs')
+        >>> plg.import_region(CirclePixelRegion(center=PixCoord(x=1163.618408203125, y=1433.47998046875), radius=141.28575134277344))  # noqa E501
+        >>> type(plg.get_regions()['Subset 2'])
+        <class 'regions.shapes.circle.CircleSkyRegion'>
+        >>> type(plg.get_regions(wrt_data='NDData[DATA]')['Subset 2'])
+        <class 'regions.shapes.circle.CirclePixelRegion'>
+
+        >>> cubeviz = Cubeviz()
+        >>> cubeviz.load_data(getfixture('spectrum1d_cube'))
+        >>> plg = cubeviz.plugins['Subset Tools']
+        >>> plg.import_region(CirclePixelRegion(center=PixCoord(x=24.27156066879736, y=22.183517455582475), radius=4.7523674964904785))  # noqa E501
+        >>> type(plg.get_regions()['Subset 1'])
+        <class 'regions.shapes.circle.CircleSkyRegion'>
+        >>> type(plg.get_regions(wrt_data='Unknown spectrum object[FLUX]')['Subset 1'])
+        <class 'regions.shapes.circle.CirclePixelRegion'>
         """
+        if return_sky_region is not None:
+            raise ValueError('return_sky_region no longer used, use wrt_data instead')
 
         if region_type is not None:
             region_type = region_type.lower()
             if region_type not in ['spectral', 'spatial']:
                 raise ValueError("`region_type` must be 'spectral', 'spatial', or None for any.")
+            elif region_type == 'spectral' and wrt_data:
+                raise ValueError('Unable to retrieve SkyRegion objects for spectral subsets')
```

Comment:
do we have a follow-up ticket for this yet?

Reply:
Top of the backlog.